Ромахина Галина

GalinaRoma opened the pull request:
[Посмотреть решение](https://rawgit.com/GalinaRoma/markup-task-4/master/index.html)

GalinaRoma committed:
Решение задачи

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

bratva commented on index.css:
```diff
@@ -0,0 +1,162 @@
+.cats
+{
+    width: 320px;
+    height: 220px;
```
Картинки получились непропорционально растянуты/сжаты. 
Так же в задании картинки разной высоты, у одной

GalinaRoma replied:
Я уберу height, согласна,что это искажает картинку
Не поняла про картинки разной высоты

GalinaRoma replied:
И еще вопрос: какие условия у меня не выполнены?

bratva commented on index.css:
```diff
@@ -0,0 +1,162 @@
+    width: 320px;
+    height: 220px;
+    display: block;
+    position: absolute;
```
зачем в данном случаи absolute не понимаю

GalinaRoma replied:
для выравнивания картинки по нижнему краю

bratva replied:
Да, это бы работало, но у тебя они все одной высоты. Т.к. смысла выравнивать там абсолютом нет

GalinaRoma replied:
Поняла

bratva commented on index.css:
```diff
@@ -0,0 +1,162 @@
+
+p
+{
+    margin: 0 auto;
```
Нет смысла, т.к. они у тебя все на 100% ширины.

bratva commented on index.html:
```diff
@@ -3,7 +3,456 @@
+        <label for="bar">Значки</label>
+        <ul>
+            <li>
+                <p class="imageCat"><img class="cats" src="images/Кузя.jpg" alt="Котенок Кузя"></p>
```
Точно не p. Посмотри семантику тега `p`.

bratva commented on index.html:
```diff
@@ -3,7 +3,456 @@
+            <li>
+                <p class="imageCat"><img class="cats" src="images/Кузя.jpg" alt="Котенок Кузя"></p>
+                <div>
+                    <p class="name"><a href="images/Кузя.jpg" title="Кузя">Кузя</a></p>
```
тоже не p

bratva commented on index.html:
```diff
@@ -3,7 +3,456 @@
+                    <p class="name"><a href="images/Кузя.jpg" title="Кузя">Кузя</a></p>
+                    <p class="category"><a href="images/Кузя.jpg" 
+                        title="Категория: кошки, рыжие">Животные, кошки, рыжие</a></p>
+                    <p class="kindness">
```
и тут не p

bratva reviewed (commented):
1. Изображения звёзд не оптимизированы. Зачем грузить такую большую картинку.
2. Вёрстка на `p`
3. Часть условий задачи не выполнена
🍅

GalinaRoma committed:
Изменила тег p в случаях, где он не подходит по значению, на div;
Сжала картинки звезочек;
Убрала у картинок статическую высоту, при этом оставила position: absolute для их расположения.

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

bratva reviewed (commented):
Сделай еще пожалуйста переключение отображения внешнего вида - иконками, а не просто радиокнопками. Остальное - хорошо
🍅

GalinaRoma committed:
Скрыла радиокнопки об отображении за картинки

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

GalinaRoma committed:
Отправила коммит, чтобы убрать ошибку загрузки

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

bratva commented:
Лучше. Но теперь по понятно, какой пункт активен =(
🍅

GalinaRoma committed:
теперь вид отображения текущий красится

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты
